pkg/server/api_v2_sql.go

```diff
@@ -597,3 +621,10 @@ func (j jsonError) MarshalJSON() ([]byte, error) {
 	pqErr := pgerror.Flatten(j.error)
 	return gojson.MarshalIndent(pqErr, "", "")
 }
+
+// This regexp is constructed from client-side queries written in
+// `databaseDetailsApi.ts` and `tableDetailsApi.ts`. It's meant to
+// filter out queries that use `SHOW RANGES` which are expensive to
+// execute and can be disabled by the operator. See the Note above for
+// an explanation of this hack.
+var localityMetadataQueryRegexp = regexp.MustCompile(`SELECT.+AS store_ids\s+FROM \[SHOW RANGES FROM (DATABASE|TABLE) .+\], unnest\(replicas\) AS unnested`)
```

Comment:
Just wondering, do we need to be this specific for the regexp? Why not just match on SHOW RANGES and an application name? I don't really mind either approach though.